app/screens/search/search.js

```diff
@@ -99,6 +106,10 @@ class Search extends PureComponent {
         }
     }
 
+    componentWillUnmount() {
+        mattermostManaged.clearListeners();
```

Comment:
Doesn't this clear all listeners? So if you open the app (to the post list), go to the search view, and then close the search view, it'll unmount the post list's listener?

Reply:
actually you are totally right, the thing is that we don't have a way to clear only one listener, I'll make the change to include that as well in this ticket